axlearn/experiments/text/gpt/c4_trainer.py

```diff
@@ -109,6 +109,9 @@ def named_trainer_configs() -> Dict[str, TrainerConfigFn]:
             )
             kwargs = fuji.get_trainer_kwargs(model_size, vocab_size=vocab_size, version=version)
             max_sequence_length = kwargs.pop("max_sequence_length")
+
+            # TODO remove before merging
+            kwargs["max_step"] = 1000
```

Comment:
how can I do this at runtime? It makes it harder to experiment when the max step is so large. I want to be able to pass this through as a command line parameter.

Reply:
You can create a separate module with your changes and point to it via `--module`, similar to your previous fuji experiments. Let me know if you prefer a more concrete example.

Reply:
yeah I know how to do it. I still don't like it. I prefer to be able to specify max_step through CLI so I can use the upstream modules for testing my PRs. I do get wanting to express everything as code so feel free to ignore this.

Reply:
While `max_step` is probably a simple case, often times trainer configs have multiple layers of composition which is not really conducive to flag overrides. Code is also more traceable/reviewable, as experiments can be tracked in commit history (and golden configs) rather than someone's bash history.